src/driver/driver_api.cc

```diff
@@ -55,6 +55,7 @@ TVM_REGISTER_PASS_CONFIG_OPTION("tir.use_async_copy", Bool);
 TVM_REGISTER_PASS_CONFIG_OPTION("tir.merge_async_commit_queue_scope", Bool);
 TVM_REGISTER_PASS_CONFIG_OPTION("tir.instrument_lwp", Bool);
 TVM_REGISTER_PASS_CONFIG_OPTION("tir.vtcm_capacity", Integer);
+TVM_REGISTER_PASS_CONFIG_OPTION("tir.ptx_pred_ldg32", Bool);
```

Comment:
the name is a bit confusing, can you discuss with @Rainy-Memory and figure out together something more comprehensible?

our key objective is that users may need to set at most one flag (zero is the best if possible) so that they could deliver the best GEMM performance out of the box